rosidl_runtime_c/docs/FEATURES.md

```diff
@@ -27,3 +27,16 @@ They are defined for the three main ROS 2 rosidl types:
 * Messages (`message_type_support_struct.h`)
 * Services (`service_type_support_struct.h`)
 * Actions (`action_type_support_struct.h`)
+
+## Type Description Interface Structs
+
+A copy of the output of `rosidl_generator_c` for `type_description_interfaces` messages.
```

Comment:
I think it is worthwhile to describe why this copy is here in the first place (to break the cycle).  Otherwise it might be hard for someone to understand why it is here.